Legacy/bonej/src/main/java/org/bonej/plugins/ParticleCounter.java

```diff
@@ -106,8 +107,8 @@ public void run(final String arg) {
 		final String units = cal.getUnits();
 		final GenericDialog gd = new GenericDialog("Setup");
 		final String[] headers = { "Measurement Options", " " };
-		final String[] labels = new String[10];
-		final boolean[] defaultValues = new boolean[10];
+		final String[] labels = new String[12];
+		final boolean[] defaultValues = new boolean[12];
```

Comment:
I think these would be clearer as
```
final String[] labels = {"Exclude on sides", ... "Skeletons"};
final boolean[] defaultValues = {false, ... false};
``` 
If most defaults are false you could do
```
final boolean[] defaultValues = new boolean[labels.length];
defaultValues[8] = true; 
```

Reply:
It's easier to add and remove items if the array index is next to the String: harder to lose count of which label belongs to which value.